home/views/add_professor.py

```diff
@@ -47,3 +46,56 @@ def post(self, request):
             }
 
         return JsonResponse(context)
+
+class EditReview(View):
+    def post(self, request):
+        user = request.user
+        review = Review.unfiltered.filter(pk=request.POST['review_id']).select_related("course").first()
```

Comment:
why not use `.get(pk=)` instead of `.filter(pk=).first`?

Reply:
At the time of writing this code I felt it was better to avoid the possibility of the review somehow not existing and `.get()` crashing the app with its `DoesNotExist` error. Ironically, I realize that I don't check that the review exists anyway so a similar issue is still possible. 

Thinking about this now, if the review doesn't exist and we get to this point, then there's a problem somewhere else. Using `.get()` would make the assumption that the review will always exist (which, at this point, it should) so I'm ok with using `.get()` here